tasks/get_current_archive_list/get_current_archive_list.py

```diff
@@ -462,20 +463,30 @@ def get_s3_credentials_from_secrets_manager(secret_arn: str) -> tuple:
     return s3_access_key, s3_secret_key
 
 
+@dataclass(frozen=True)
+class MessageData:
+    """
+    aws_region: (str) The region the report bucket resides in
```

Comment:
Based on your code, should be `report_bucket_region` instead of `aws_region`.

If you use `Args:`, should space theses in all 4

Reply:
Whoops. Fixed.